packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearchUtils.ts

```diff
@@ -53,8 +52,8 @@ export function useTokensFromUser(): SearchableTokenStorage {
       const bridgeToken = bridgeTokens[_address]
 
       // Any tokens in the bridge that don't have a list id were added by the user.
-      if (bridgeToken && !bridgeToken.listID) {
-        storage[_address] = { ...bridgeToken, tokenLists: [] }
+      if (bridgeToken && !bridgeToken.listIds.size) {
```

Comment:
same comment, if possible please use `size === 0` unless i misunderstood your intention, or maybe use a function called `isEmptySet` that returns a boolean